src/main/java/uk/gov/hmcts/divorce/ccd/search/CaseFieldsConstants.java

```diff
@@ -11,7 +11,7 @@
     public static final String DIVORCE_COSTS_CLAIM = "divorceCostsClaim";
     public static final String DIVORCE_OR_DISSOLUTION = "divorceOrDissolution";
     public static final String FINANCIAL_ORDER = "financialOrder";
-
+    public static final String PETITIONER_ORGANISATION_POLICY = "petitionerOrganisationPolicy";
```

Comment:
wouldn't it be petitionerSolicitorOrganisationPolicy? Also shouldn't petitioner now be applicant?

Reply:
as discussed Tom is already having a separate PR to update naming conventions